npm/pkg/dataplane/policies/chain-management_linux.go

```diff
@@ -130,16 +135,31 @@ func (pMgr *PolicyManager) initializeNPMChains() error {
 // removeNPMChains removes the jump rule from FORWARD chain to AZURE-NPM chain
 // and flushes and deletes all NPM Chains.
 func (pMgr *PolicyManager) removeNPMChains() error {
+	// 1.1 delete the deprecated jump rule from FORWARD chain to AZURE-NPM chain, if it exists
+	deprecatedDeleteErrCode, deprecatedDeleteErr := pMgr.runIPTablesCommand(util.IptablesDeletionFlag, deprecatedJumpFromForwardToAzureChainArgs...)
```

Comment:
Does this mean we will suppress any other kind of error without log ? we should at least log it!

Reply:
Oh, I meant to do the negated version of that boolean. Updating to do a switch statement for all possible error codes